.npmrc

```diff
@@ -0,0 +1 @@
+registry = "https://registry.npmjs.com/"
```

Comment:
This is mostly because of my local npm configuration. We have our own internal npm registry at Square that this prevents us from using. And yeah, this is the same as yarnpkg.com for now. I'll update this to use that one.